env/CMakeLists.txt

```diff
@@ -36,15 +35,15 @@ ExternalProject_Add(
 # -DMLIR_ENABLE_EXPENSIVE_PATTERN_API_CHECKS=ON set if debug
 ExternalProject_Add(
     llvm-project
-    PREFIX ${TTMLIR_TOOLCHAIN_DIR}
     # Super hacky way to install the python dependencies before the build
-    PATCH_COMMAND bash -c "source ${CMAKE_CURRENT_SOURCE_DIR}/activate && pip install -r ${TTMLIR_TOOLCHAIN_DIR}/src/llvm-project/mlir/python/requirements.txt"
+    PATCH_COMMAND bash -c "source ${CMAKE_CURRENT_SOURCE_DIR}/activate && pip install -r mlir/python/requirements.txt"
     CMAKE_GENERATOR Ninja
     CMAKE_ARGS
     -DPython3_FIND_VIRTUALENV=ONLY
     -DPython3_EXECUTABLE=${TTMLIR_TOOLCHAIN_DIR}/venv/bin/python
     -DCMAKE_INSTALL_PREFIX=${TTMLIR_TOOLCHAIN_DIR}
-    -DLLVM_ENABLE_PROJECTS=mlir
+    # Projects: bolt;clang;clang-tools-extra;compiler-rt;cross-project-tests;libc;libclc;lld;lldb;mlir;openmp;polly;pstl;flang
+    -DLLVM_ENABLE_PROJECTS=mlir;clang;clang-tools-extra
```

Comment:
Do we really want to build clang and clang tools?  This will make the image bigger / take longer to build an image.

Reply:
I added this because we were using "clang-tidy" and it gets installed through "clang-tools-extra".
Maybe "clang-tools-extra" can build with "clang", I'll try it out

Reply:
Works fine without clang and tools 👍